### ISSUE TITLE
AU Accessible Version: NVDA exits focus mode with "Academics" nav menu option

### ISSUE THREAD

**Issue by dcmcnabb** [terrill/AU#4]
**Environment:**

https://www.washington.edu/accesscomputing/AU/after.html

- NVDA 2019.2 / Firefox 68 / Windows 10

**Steps to Reproduce:**
1. Go to https://www.washington.edu/accesscomputing/AU/after.html
2. Start NVDA
3. Use the Tab key to focus on "Academics" (au_after_academics.jpg)

**Expected Result:**

- NVDA switches to "focus" mode when it enters the menu
- NVDA remains in focus mode when the focus moves to "Academics"

**Actual Result:**

When the focus moves to "Academics", NVDA exits "focus" mode.  The &lt;a href&gt; containing the menu item doesn't have a role="menuitem" attribute.

au_after_academics.jpg:
![au_after_academics](https://user-images.githubusercontent.com/20907097/63549826-d66d5880-c4f6-11e9-9ec1-1775869dfd85.jpg)

**Comment by terrill** [comment]
Thanks @dcmcnabb ! This has been fixed in c907c966326f6d34e6758a74add66b9ab08c9f04
